crates/ironclaw_webui_v2_static/static/js/i18n/ar.js

```
@@ -657,9 +662,9 @@ registerPack("ar", {
   "automations.summary.active": "نشطة",
   "automations.summary.activeDetail": "الجداول الزمنية الممكّنة في انتظار التشغيل التالي.",
   "automations.summary.running": "Running now",
-  "automations.summary.runningDetail": "Runs currently in progress.",
+  "automations.summary.runningDetail": "الأتمتات التي لديها تشغيل قيد التقدم.",
```

Biome (2.4.16)

[error] This property is later overwritten by an object member with the same name.

(lint/suspicious/noDuplicateObjectKeys)